ansible/roles/contiv/tasks/netmaster.yml

```diff
@@ -0,0 +1,29 @@
+
+- name: copy netmaster exec file
+  copy: src={{ contiv_bin_path }}/netmaster dest=/usr/bin/netmaster mode=0755
```

Comment:
We can't have /usr/bin/ hardcoded b/c /usr/bin/ is RO on coreos. Can you follow the existing design pattern for copying bins?